docs/elements.md

```diff
@@ -1865,6 +1866,39 @@ Note that only one of the attributes `source-file-name`, `submitted-file-name` o
 
 - [`pl-file-editor` to provide an in-browser code environment](#pl-file-editor-element)
 
+### `pl-card` element
+
+Displays question content within a card-styled component. Optionally displays content in a header, footer, or image caps via tag attributes.
+
+#### Sample element
+
+```html
+<pl-card header="Header" title="Title" width="50%" img-bottom-src="https://picsum.photos/536/354">
+  <pl-question-panel> This card is 50% width and has a bottom image cap. </pl-question-panel>
+</pl-card>
+```
+
+#### Customizations
+
+| Attribute        | Type   | Default | Description                             |
+| ---------------- | ------ | ------- | --------------------------------------- |
+| `header`         | string | -       | Contents of the card header             |
+| `title`          | string | -       | Contents of the card title              |
+| `subtitle`       | string | -       | Contents of the card subtitle           |
+| `contents`       | string | -       | Raw contents of the card body           |
+| `footer`         | string | -       | Contents of the card footer             |
+| `img-top-src`    | string | -       | Source URL for the top image cap        |
+| `img-bottom-src` | string | -       | Source URL for the bottom image cap     |
+| `width`          | string | auto    | Width of the card (25%, 50%, 75%, auto) |
+
+#### Details
+
+The `pl-card` attributes mirror the options of [Bootstrap 4 cards](https://getbootstrap.com/docs/4.6/components/card/).
+
+#### Example implementations
+
+- [element/card]
```

Comment:
I think you have to add a link to this at the bottom of the page for this to work.

Reply:
Fixed in 24c4449c802923ab6285eee8c528458967f6c47e